src/library-authoring/common/LicenseField.jsx

```diff
@@ -60,11 +60,10 @@ export const LicenceFieldBase = (
         </div>
         <Row className="border-bottom py-2">
           <Col xs={1} className="text-center align-self-center">
-            <Input
-              type="checkbox"
+            <Form.Checkbox
               id="attribution"
               name="attribution"
-              className="m-0 p-0 position-relative"
+              // className="m-0 p-0 position-relative"
```

Comment:
is there a plan to bring these back at some point? if not let's just remove the line instead of commenting it out

Reply:
let me update it as well